File: blog-post-2/programjava/LeapYear.java
package programjava;

import java.util.Scanner;

public class LeapYear {
    public static void main(String[] args) {
        boolean isLeapYear = false;
        Scanner in = new Scanner(System.in);
        System.out.print("Enter year: ");
        int year = in.nextInt();
        in.close();
        
        if(year % 4 == 0){
            isLeapYear = true;
            if(year % 100  == 0 && year % 400 != 0 ){
                isLeapYear = false;
            }
        }

        if(isLeapYear){
            System.out.println("leap year");
        }
        else{
            System.out.println("not leap year");
        }
    }
}
//create tax clculator

File: blog-post-2/kotlin/LeapYear.kt
import java.util.Scanner

fun main(args: Array<String>) {
    var isLeapYear = false;
        var scanner = Scanner(System.`in`)
        print("Enter year: ")
        var year = scanner.nextInt()
        scanner.close()
        
        if(year % 4 == 0){
            isLeapYear = true
            if(year % 100  == 0 && year % 400 != 0 ){
                isLeapYear = false
            }
        }

        if(isLeapYear){
            System.out.println("leap year")
        }
        else{
            System.out.println("not leap year")
        }
}